docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,607 @@
 ---
-title: Teleport Trusted Clusters
-description: How to configure access and trust between two Teleport Clusters.
-h1: Trusted Clusters
+title: Configure Trusted Clusters
+description: Explains how you can configure a trust relationship and manage access between two Teleport clusters.
 ---
 
-Teleport can partition compute infrastructure into multiple clusters. A cluster
-is a group of resources connected to a Teleport Auth Server. Each cluster
-manages a separate certificate authority (CA) for all users and resources.
-
-Trusted Clusters allow the users of one cluster, the **root cluster**, to
-seamlessly connect to resources in another cluster, the **leaf cluster**, while
-remaining authenticated with only a single Auth Service. The leaf cluster can
-run behind a firewall without any ingress ports open.
+As you learned in [Core Concepts](../../core-concepts.mdx), a Teleport cluster
+consists of the Teleport Auth Service, the Teleport Proxy Service, and the 
+Teleport services that manage access to resources in your infrastructure.
+With Teleport, you can also partition your infrastructure into multiple connected 
+clusters to allow the users of one cluster—the **root cluster**—to connect to resources 
+in another cluster—the **leaf cluster**—while authenticated using a single 
+Teleport Auth Service. 
+
+After you establish a trust relationship between the root and leaf clusters, the leaf cluster can
+run behind a firewall without any inbound ports open. The leaf cluster creates an outbound reverse 
+SSH tunnel to the root cluster and keeps the tunnel open.
+When a user tries to connect to any resource in the leaf cluster, the Teleport Auth Service on the 
+leaf cluster connects to the root cluster through the reverse tunnel using the Teleport Proxy Service 
+instance running on the root cluster.
+
+## Who uses trusted clusters?
+
+Most organizations don't need to configure trusted clusters. In most cases, you can add 
+reverse-tunnel proxy instances and enroll and manage thousands of resources without creating a new 
```

Comment:
```suggestion
Teleport Proxy Service instances to enroll and manage thousands of resources without creating a new 
```
To keep architectural terms consistent

Reply:
1) Put the `jq` look up commands inside of the related steps. Sort of ugly.
2) I don't think adding "This guide . . ."  or "This article..." to the beginning of the description adds any value. It's pretty common to start sentences with verbs.
3) Yes, I'll change the Core concepts usage. I think Teleport Database Service is fine. But Database Service isn't any clearer than database service. If we need to make distinctions, using a full proper name is fine but trusted and cluster are just common words and you can easily overwhelm the reader with too many capitalized words 
4) 👍